.gitignore

```diff
@@ -88,3 +88,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# PyCharm
```

Comment:
I'm curious, what are your code-style settings in PyCharm?

I'm currently using the [default PEP8 checks](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) in my code, and, as far as I remember the "max line length" in PyCharm is not set to 79? 

I might add some documentation about this.

I'm aware that a short line length can sometimes be super annoying :(